nvflare/app_opt/xgboost/tree_based/executor.py

```diff
@@ -34,7 +34,8 @@
 class FedXGBTreeExecutor(Executor):
     def __init__(
         self,
-        training_mode,
+        split_mode,
+        secure_training,
         lr_scale,
```

Comment:
please revert changes to this file, this file is using "training_mode" for differentiating "cyclic" vs "bagging"

Reply:
Reverted.